ehb_service/apps/core/models/identities.py

```diff
@@ -373,10 +373,11 @@ def responseFieldDict(self):
         return response
 
     def __unicode__(self):
-        return "{0}, {1} IN {2}".format(
+        return "{0}, {1} IN {2} ID {3}".format(
```

Comment:
It looks like 'ID {3}' was added by accident. please remove

Reply:
ec3aa9077077640eefcc2280e0c19c1d414eaaff